lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js

```diff
@@ -21,6 +21,21 @@ const Gatherer = require('../gatherer');
 /* global document,window */
 
 /* istanbul ignore next */
+function saveAsyncLinks() {
+  function checkForLinks() {
+    document.querySelectorAll('link').forEach(link => {
+      if (link.rel === 'preload' || link.disabled || link.async) {
```

Comment:
Though I'm not sure we need to check for this one. `async` only applies to HTMLImports and I can think of a use case where someone would set `async` on an import, only to remove it later.